### PR TITLE
Indent multi string literal to column 1

### PR DESCRIPTION
**Pull request by spuppala-rythmos**
Introduced option indent_col1_multi_string_literal

if true multi string literal will be indented to column 1

reverted the test cases to old as we introduced the option.

### CODE REVIEW

File: tests/output/cs/10090-string_multi.cs [hunk 1]
```diff
@@ -7,7 +7,7 @@ public void F()
       var y = @"
 abc" + "def";
       var z = "" +
-@"
+              @"
```
Comment (hschlichter) [inline tests/output/cs/10090-string_multi.cs:10]: Doesn't this sort of contradict what the test in indent-multistring-coulmn1.cs says?

I would not expect
```cs
var z = "" +
@"
";
```
to be formatted to this
```cs
var z = "" +
            @"
";
```

**Review by hschlichter — CHANGES_REQUESTED** [review]
The changed tests in 10090-string_multi.cs looks weird and seems to contradict other tests. 

Might be I'm just misunderstanding something, but some clarification would be nice :-)

File: tests/output/cs/10090-string_multi.cs (hunk 1 above)
Comment (spuppala-rythmos) [reply tests/output/cs/10090-string_multi.cs:10]: Hi @hschlichter 

Now we have added an option and not enabled the same. so you are getting those diffs.
last time you have merged (mistakenly due to some diff issue) the valid and invalid diffs. As a result now those valid also reverted and looks contradict)

if you merge this PR all valid and invalid will be reverted and then later we can create new PR with enabling option in our config then we can see all the valid diffs

let me know if you have any question (i should have explained this before sorry for that :) )

Also regarding 10090 it doesn't have config option enabled so the it looks weird. (not added option as the test case is coming from main master), enabled the option only for unity specific test cases in staging.

Comment (spuppala-rythmos) [reply tests/output/cs/10090-string_multi.cs:10]: @hschlichter this one we discussed int the call and agreed to accept the PR

once this is closed we will create new PR enabling option

**Review by hschlichter — APPROVED** [review]

File: tests/output/cs/10090-string_multi.cs (hunk 1 above)
Comment (mihaipopescu) [reply tests/output/cs/10090-string_multi.cs:10]: would actually prefer to have the multiline string on column 1

**Comment by mihaipopescu** [comment]
@satishp87 any new updates on this PR?

File: tests/output/cs/10090-string_multi.cs (hunk 1 above)
Comment (spuppala-rythmos) [reply tests/output/cs/10090-string_multi.cs:10]: @mihaipopescu once this PR is merged then i can create one proper PR which gives all proper and expected results ... a big story explained above which is little bit difficult to understand 😨

**Comment by hschlichter** [comment]
Diff has been added to jenkins.

Test this please.

**Comment by hschlichter** [comment]
test this please

**Comment by mihaipopescu** [comment]
@satishp87 one test seems to be failing though `25 - c-sharp_10074 (Failed)`

**Comment by mihaipopescu** [comment]
test this please

**Comment by mihaipopescu** [comment]
test this please

**Pull request merged**